src/firestore/util.ts

```diff
@@ -162,6 +198,17 @@ export const initializeFirebaseProject = async (
     };
   } else {
     const app = safeInitializeApp(config as LiveFirebaseConfig, name);
+
+    // Initialize App Check with reCAPTCHA provider before calling any other Firebase services
+    // Grab the App Check token for use in the ROAR Dashboard Axios Calls to Firebase
+    const appCheck = initializeAppCheckWithRecaptcha(app, name, siteKey, debugToken);
```

Comment:
Can we do this conditionally, based upon whether or not the `siteKey` and `debugToken` are defined?